src/py/CMakeLists.txt

```diff
@@ -23,14 +23,24 @@
 #####################################################################################
 
 option(MIGRAPHX_ENABLE_PYTHON "Enable python bindings" ON)
+add_library(migraphx_py py_loader.cpp)
```

Comment:
can this be named differently? It already has `migraphx_py_${PYTHON_VERSION}` later for `py.cpp`

Reply:
No, its named accordingly. `migraphx_py` is the library you would link, which will load the corresponding `migraphx_py_${PYTHON_VERSION}` depending which python version is installed on the system.